content/supported-blockchains/_index.md

```diff
@@ -16,47 +16,62 @@ Pocket can be made to work with any network/blockchain (RelayChain) that uses th
-| Polygon Mumbai               | polygon-mumbai          | 000F         |
-| Solana                       | solana-mainnet          | 0006         |
-| Starknet Mainnet             | starknet-mainnet        | 0060         |
-| Starknet Testnet             | starknet-testnet        | 0061         |
-| Swimmer Network Mainnet      | avax-cra                | 03CB         |
-| Velas                        | velas-mainnet           | 0067         |
-| Velas Archival               | velas-mainnet-archival  | 0068         |
+| Name                                        | Portal API Prefix         | RelayChainID |
+| ------------------------------------------- | ------------------------- | ------------ |
+| Algorand Archival                           | algo-archival             | 000D 
+| Algorand Mainnet                            | algo-mainnet              | 0029 
+| Arbitrum One                                | arbitrum-one              | 0066 
+| AVAX                                        | avax-mainnet              | 0003 
+| AVAX Archival                               | avax-archival             | 00A3 
+| AVAX_CRA                                    | avax-cra                  | 03CB 
+| AVAX_DFK                                    | avax-dfk                  | 03DF 
+| BOBA Mainnet                                | boba-mainnet              | 0048 
+| Binance Smart Chain                         | bsc-mainnet               | 0004 
+| Binance Smart Chain (Archival)              | bsc-archival              | 0010
+| Celo Mainnet                                | celo-mainnet              | 0065 
+| Dogechain Mainnet                           | dogechain-mainnet         | 0059
+| Ethereum Mainnet                            | eth-mainnet               | 0021 
+| Ethereum Mainnet Archival                   | eth-archival              | 0022
+| Ethereum Mainnet Archival with trace calls  | eth-trace                 | 0028
+| Ethereum Mainnet High Gas                   | ethereum-mainnet-high-gas | 0062
+| Evmos Mainnet                               | evmos-mainnet             | 0046 
+| Fantom Mainnet                              | fantom-mainnet            | 0049 
+| Fuse                                        | fuse-mainnet              | 0005 
+| Fuse Archival                               | fuse-archival             | 000A 
+| Gnosis Chain Archival                       | poa-xdai-archival         | 000C
+| Gnosis Chain Mainnet                        | poa-xdai                  | 0027 
+| Goerli                                      | eth-goerli                | 0026 
+| Goerli Archival                             | goerli-archival           | 0063 
+| Harmony Shard 0                             | harmony-0                 | 0040 
+| IoTeX Mainnet                               | iotex-mainnet             | 0044 
+| KAVA Mainnet Archival                       | kava-mainnet-archival     | 0072
```

Comment:
Capital-case for Kava, and put the Archival _below_ the mainnet one.

Reply:
done 👍